android/build.gradle

```diff
@@ -2,30 +2,30 @@ group 'io.github.zeshuaro.google_api_headers'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.5.30'
     repositories {
         google()
-        jcenter()
+        mavenCentral()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.0'
+        classpath 'com.android.tools.build:gradle:7.1.2'
```

Comment:
Do we need this change? Latest Flutter still generates it with `4.1.0`.

Reply:
Yes the Flutter one is outdated. Gradle 7 brings lots of fixes like native building on M1 macbooks, and is also required for gradle-7.4.1